tests/conftest.py

```diff
@@ -202,22 +222,27 @@ def application_process(
 
 
 @fixture(scope="session")
-def docker_client() -> DockerClient:
-    return docker.from_env()
+def docker_client(tests_id: str) -> Generator[Tuple[DockerClient, str], None, None]:
```

Comment:
Changing this fixture is a great clutter in the code, now all code needs to use `docker_client[0]` which is not convenient. This is too many changes in the tests. It makes no sense to change all sites + require all sites inn the future to use a tuple and extract the value.

Options I can think of:
* Extend the `DockerClient` class to one which implements the functionality (e.g accepts the label in its c'tor and adds the label for all created containers)
* Apply the change only in the `start_container` method, if it's the only method calling `container.run`. Need to find a way to pass the value to it without modifying ALL tests

Reply:
True, I was thinking about saving the test_id to a local file and reading it there :)

But I like the idea about extended class better.